packages/core/src/plugins/web3FormAssets/web-3-form.css

```diff
@@ -0,0 +1,18 @@
+.form-container {
+    border-radius: 5px;
+    background-color: #00b0f0;
```

Comment:
I'm guessing this background colour is meant to mirror the `primary` type box's background colour (correct me if I'm wrong!). Is it possible to apply this using the `box` container instead of hardcoding it in the CSS? This would have the benefit of being explicit about the link between the `primary` theme and this form's default colour, and also avoid future issues if there are changes in the Bootstrap themes (e.g. Bootstrap migrations).

Reply:
Sorry removed! I did this before moving over to markbind's box :sweat_smile: